index.html

```diff
@@ -5,77 +5,151 @@
 	<meta charset=utf-8>
 	<title>Wireguard Key Generator (Trustless)</title>
 	<meta name="author" content="Juan I Carrano">
-	<script src="wireguard.js"
-		integrity="sha256-KnkrBLei0lXuE9dGbMURxP000Jg7a9Tm8Gvvc6DysrU"
+	<script src="wireguard.js" integrity="sha256-KnkrBLei0lXuE9dGbMURxP000Jg7a9Tm8Gvvc6DysrU="
 		crossorigin="anonymous"></script>
 	<style>
-		body{margin:40px
-		auto;max-width:650px;line-height:1.6;font-size:18px;color:#444;padding:0
-		10px}h1,h2,h3{line-height:1.2}
-		h2 {font-size:large;}
-		div {padding: 1ex; margin-top: 1ex;}
-		div.adm {background-color: rgb(255, 231, 231);}
-		div.usr {text-align: center;}
-		#gen {background-color: rgb(248, 206, 69);}
-		#results {display: none; background-color: rgb(91, 241, 136);}
+		body {
+			margin: 40px auto;
+			max-width: 650px;
+			line-height: 1.6;
+			font-size: 18px;
+			color: #444;
+			padding: 10px
+		}
+
+		h1,
+		h2,
+		h3 {
+			line-height: 1.2
+		}
+
+		h2 {
+			font-size: large;
+		}
+
+		div {
+			padding: 1ex;
+			margin-top: 1ex;
+		}
+
+		div.adm {
+			background-color: rgb(255, 231, 231);
+		}
+
+		div.usr {
+			text-align: center;
+		}
+
+		div.usr a {
+			text-decoration: none;
+		}
+
+		#gen {
+			background-color: rgb(248, 206, 69);
+		}
+
+		.disabled {
```

Comment:
The button is already greyed out by the browser (and this would select the class anyways). Better use this to grey out the text input:

```suggestion
		[readonly] {
```